### ISSUE TITLE
Bi-Directional RPC for the Web -- A possible feature/porting (maybe a possible example)

### ISSUE THREAD

**Issue by arkenidar** [eleme/thriftpy#176]
TLDR I cannot find the way to port this code:
http://omri.org.il/2012/12/22/python-tornado-apache-thrift-and-javascript/
to work from thrift to thriftpy.
Bi-Directional RPC for the Web -- A possible feature (with relative example)
Hi there! I was searching for bi-directional RPC between a Web Browser CLIENT and a Pythonic SERVER (possibly over Web Sockets) ... and I found this article:
http://omri.org.il/2012/12/22/python-tornado-apache-thrift-and-javascript/
I am unable for the moment to adapt it to thriftpy, but when done it could be a good example of the desired feature (Bi-Directional RPC for the Web)
Is this already available? Which samples, documentation, or other material should I read?
Thanks!
PS Happy New Year 2016
